release/README.md

```diff
@@ -1,29 +1,28 @@
-## Intel Data Center GPU Driver Container Images For OpenShift Release
-### General
-Intel data center GPU driver container images are certified and released on the [Red Hat Ecosystem catalog](https://catalog.redhat.com/). To make the driver container images managed by KMM operator easy to use and track, the following naming convention is adopted.
+# Intel® Data Center GPU Driver for OpenShift*  Release Information
 
```

Comment:
I think we don't need ® and * here, because in the root page it is already added.

Reply:
Done..